### ISSUE TITLE
hello.byte doesn't work

### ISSUE THREAD

**Issue by chenglou** [andreypopp/esy-ocaml-project#5]
After `esy install && esy build`:

```
❯ ./hello.byte
Fatal error: unknown C primitive `lwt_unix_bind_job'
```

We should be doing `esy ocamlrun hello.byte`, but this isn't obvious for newcomers.

**Comment by andreypopp** [comment]
Try `esy ./hello.byte`, it needs a proper env

**Comment by chenglou** [comment]
That has the overhead of calling esy, right?

**Comment by andreypopp** [comment]
Yes, this has to be fixed though — by reusing the cached env if available.

**Comment by chenglou** [comment]
But fetching from the cache also has an overhead right? Especially if we start up node in the process

**Comment by andreypopp** [comment]
The plan is to make this entirely in bash, without starting node.

**Comment by chenglou** [comment]
What about windows? Apart from the new bash stuff

**Comment by andreypopp** [comment]
I think we can get the there but it wasn't my focus recently so really
can't comment on that. The latest try to get this running on win was
stumbled upon non availability of opam-installer-bin package for the
platform.

**Comment by chenglou** [comment]
Hummm k. I'm asking because for the js compilation story, we're making reason work on Windows and that'd have been the last piece left. If we embark on esy then it'd be adding yet another windows-incompatible thing. We can worry about this later I guess.

Bash startup is instant right? My zsh with all the configs starts up slower than node

**Comment by andreypopp** [comment]
Yeah, bash startup is pretty fast. We also make it not read profile and
bashrc so that env is "pure".
